### ISSUE TITLE
ebay-infotip: modal should have label on dialog element

### ISSUE THREAD

**Issue by mkchang** [eBay/ebayui-core#1219]
# Bug Report

## Description

The new modal option for infotip does not have a way to apply a label to the element with `role="dialog"`. No input to `ebay-infotip` itself, and no input from `ebay-infotip` to the `ebay-dialog-base` to apply that label.

It looks like a `ariaLabelledby` gets generated on the dialog, but it's not associated with anything in my app at least.

## Screenshots
![image](https://user-images.githubusercontent.com/6880917/91917972-5a176480-ec76-11ea-94b6-6071e166d06d.png)

**Comment by agliga** [comment]
The aria-labelledby is for header (however this doesn't have a header)
You can use `a11yCloseText` for the close button. 
What are you looking for exactly to add? I can add an attribute to pass as aria-label or aria-labelledby to dialog if needed.

**Comment by ianmcburnie** [comment]
When the dialog lacks a visible heading for `aria-labelledby`, I wonder if we can reuse the same `aria-label` text that is used on the infotip button itself as the `aria-label` for the dialog?

**Comment by agliga** [comment]
Yes that is easy to do.
I did add a fix just now to remove the `aria-labelledby` if there is no heading so we can use that too.

**Comment by ianmcburnie** [comment]
Fixed in 5.7.0. Closing.
